data/migrations/V0045.1__public_auditsearch_grant.sql

```diff
@@ -0,0 +1,63 @@
+
+SET search_path = auditsearch, pg_catalog;
+
+CREATE OR REPLACE VIEW cmte_audit_vw AS
+SELECT dc.cmte_id,
+    dc.cmte_nm,
+    dc.fec_election_yr,
+    dc.cmte_dsgn,
+    dc.cmte_tp,
+    b.FILED_CMTE_TP_DESC::text AS cmte_desc
+FROM auditsearch.audit_case aa, disclosure.cmte_valid_fec_yr dc, staging.ref_filed_cmte_tp b
+WHERE aa.cmte_id = dc.cmte_id
+  AND dc.cmte_tp IN ('H','S', 'P', 'X', 'Y', 'Z', 'N', 'Q', 'I', 'O')
+  and dc.cmte_tp = b.filed_cmte_tp_cd
+ORDER BY dc.cmte_nm, dc.fec_election_yr;
+
+ALTER TABLE cmte_audit_vw OWNER TO fec;
+
+GRANT ALL ON TABLE cmte_audit_vw TO fec;
+GRANT SELECT ON TABLE cmte_audit_vw TO fec_read;
+GRANT SELECT ON TABLE cmte_audit_vw TO openfec_read;
+
+
+GRANT ALL ON TABLE cand_audit_vw TO fec;
+GRANT SELECT ON TABLE cand_audit_vw TO fec_read;
+GRANT SELECT ON TABLE cand_audit_vw TO openfec_read;
+
+GRANT ALL ON TABLE finding_vw TO fec;
+GRANT SELECT ON TABLE finding_vw TO fec_read;
+GRANT SELECT ON TABLE finding_vw TO openfec_read;
+
+GRANT ALL ON TABLE finding_rel_vw TO fec;
+GRANT SELECT ON TABLE finding_rel_vw TO fec_read;
+GRANT SELECT ON TABLE finding_rel_vw TO openfec_read;
+
+
+SET search_path = public, pg_catalog;
```

Comment:
This line (and the space around it) is just extra from the other migration that was there, it's not needed again since it's at the top of the file. :-)

Reply:
the top line is different schema: SET search_path = auditsearch, pg_catalog;
so I think we need this line ? right ? for public schema.
Thanks

Reply:
Oh good point, you're right, the operations are working against different schemas.  Please disregard! :-)